server/store/session.go

```diff
@@ -80,16 +79,18 @@ func (s sessionStore) listKeysForApp(appID apps.AppID) ([]string, error) {
 	ret := make([]string, 0)
 
 	for i := 0; ; i++ {
-		keys, err := s.conf.MattermostAPI().KV.ListKeys(i, ListKeysPerPage, pluginapi.WithPrefix(appKey(appID)))
```

Comment:
Why is this not correct? (Besides the check ` len(keys) < ListKeysPerPage` being an issue)

Reply:
I suppose it can come back if the check is done right, but why? I see little value in the wrapper, TBH.